tarantool/src/main/java/com/yahoo/ycsb/db/TarantoolClient.java

```diff
@@ -81,12 +79,11 @@ public Status insert(String table, String key, Map<String, ByteIterator> values)
   public Status read(String table, String key, Set<String> fields, Map<String, ByteIterator> result) {
     try {
       List<String> response = this.connection.select(this.spaceNo, 0, Arrays.asList(key), 0, 1, 0);
-      result = tupleConvertFilter(response, fields);
+      tupleConvertFilter(response, fields, result);
       return Status.OK;
-    } catch (TarantoolException exc) {
-      LOGGER.log(Level.SEVERE, "Can't select element", exc);
-      return Status.ERROR;
-    } catch (NullPointerException exc) {
+    } catch (Exception exc) {
```

Comment:
Should be `TarantoolException | NullPointerException exc`, because we want to catch the most specific one according to Java conventions and recommendations

Reply:
How about removing `catch NullPointerException` and leave any RuntimeExceptions to machinery higher on the stack?